### PR TITLE
Added policies page, fixed up footer links, added universal page title

### PR DESCRIPTION
**Pull request by minglethepringle**

### CODE REVIEW

File: v2.0/src/Components/Pages/PoliciesPage/PoliciesPage.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import CONST from '../../Constants';
+import LoadingCircle from '../../Shared/LoadingCircle';
+import PageTitle from '../../Shared/PageTitle';
+import Accordion from 'react-bootstrap/Accordion';
+import Card from 'react-bootstrap/Card';
+
+class PoliciesPage extends React.Component {
+    constructor(props) {
```
Comment (Opoku-Agyemang) [inline v2.0/src/Components/Pages/PoliciesPage/PoliciesPage.js]: I think we are now using redux to manage out state.  Or you and Kieran have agreed to let Kieran handle the changing of all the states to Redux?

File: v2.0/src/Components/Pages/PoliciesPage/PoliciesPage.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                pageData: myJson.pageData
+            });
+        }).catch(error => {
+            console.log(error);
+            return null;
+        });
+    }
+
+    render() {
+        if(!this.state.pageData) return <LoadingCircle/>;
```
Comment (Opoku-Agyemang) [inline v2.0/src/Components/Pages/PoliciesPage/PoliciesPage.js]: I like this!

File: v2.0/src/Components/Pages/ServicesPage/ServicesPage.js [hunk 3]
```diff
@@ -50,7 +50,7 @@ class ServicesPage extends React.Component {
         return vendors.map((vendor) => {
             return (
                 <div className="col-12 col-md-6 col-lg-4" key={vendor.vendor}>
-                    <div className="card rounded-0">
+                    <div className="card rounded-0 spacing">
```
Comment (Opoku-Agyemang) [inline v2.0/src/Components/Pages/ServicesPage/ServicesPage.js]: I think uploading a picture of the page will make it easier to see how things look with these style specific changes

File: v2.0/src/Components/Pages/PoliciesPage/PoliciesPage.js (hunk 1 above)
Comment (minglethepringle) [reply v2.0/src/Components/Pages/PoliciesPage/PoliciesPage.js]: Waaat I didn't realize anything about redux lol @ki3ranoday can you handle this

**Pull request merged**